sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -658,8 +705,9 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
         vlines_ravel = self.deciles_vlines_.ravel(order='C')
         hlines_ravel = self.deciles_hlines_.ravel(order='C')
 
-        for i, axi, fx, pd_result in zip(count(), self.axes_.ravel(),
-                                         self.features, self.pd_results):
+        for pd_plot_idx, (axi, feature_idx, pd_result) in enumerate(
+            zip(self.axes_.ravel(), self.features, self.pd_results)
+        ):
```

Comment:
I think when I was writing this, I thought that zip + count looked a little nicer than enumerate + zip + `()`.

Either way I am okay with it.

Reply:
it avoids an import :)